README.md

```diff
@@ -262,11 +263,12 @@ Configuration is done by user application: it should configure exporter and may
     {
         builder
             .SetSampler(Samplers.AlwaysSample)
-            .UseZipkin(o => o.ServiceName = "my-service")

```

Comment:
should we remove ServiceName from zipkin (and possibly other exporters) options?

Reply:
I didn't remove it to allow to set up the "default" service name. I agree though I'd rather switch to resource model all together